### DISCUSSION TITLE
python对接open-api回滚接口

### DISCUSSION THREAD

**Discussion by jasonqiangg** [apolloconfig/apollo#4255] [Q&A]
调用http://{portal_address}/openapi/v1/envs/{env}/releases/{releaseId}/rollback接口进行回滚操作，首先通过调用release接口(http://{portal_address}/openapi/v1/envs/{env}/apps/{appId}/clusters/{clusterName}/namespaces/{namespaceName}/releases/latest)获取发布ID，但该接口不返回releaseId,无法调用回滚接口。从数据库中拿到查询到releaseId，然后调用回滚接口，报错为{
    "timestamp": "2022-03-07T13:58:52.136+0800",
    "status": 405,
    "error": "Method Not Allowed",
    "exception": "org.springframework.web.HttpRequestMethodNotSupportedException",
    "message": "Request method 'PUT' not supported",
    "path": "/openapi/v1/envs/DEV/releases/12005/rollback"
}方法不支持，经查token没有问题，apollo版本号为1.8.0。望求解，谢谢。

**Comment by Anilople** [comment]
see https://www.apolloconfig.com/#/zh/usage/apollo-open-api-platform?id=_45-405-method-not-allowed

Could reproduce it in http://106.54.227.205?
